test-forms/index.json

```diff
@@ -496,7 +496,7 @@
   { "html": "ua_ctcorpmpc_com-signup.html" },
   { "html": "joybird_com.html" },
   { "html": "ikea_signup-verification.html", "expectedSubmitFalsePositives": 2 },
-  { "html": "chase_2fa-verification-step.html" },
+  { "html": "chase_2fa-verification-step.html", "expectedFailures": ["unknown"] },
```

Comment:
This is a big one, hard to filter it out as it very likely resembles a username in every way.